docs/source/alfacase/api_reference.rst

```diff
@@ -32,8 +32,17 @@ Outputs
 
 .. autoclass:: alfasim_sdk.ProfileOutputDescription()
 
```

Comment:
Also please add a detailed changelog entry describing the change.

Reply:
Just added it